tests/library/test_metrics.py

```diff
@@ -698,7 +699,19 @@ def test_kendalltau(self):
         outputs = apply_metric(
             metric=metric, predictions=predictions, references=references
         )
-        global_target = 0.81649658092772
+        global_target = 0.6025716662406921
+        self.assertAlmostEqual(global_target, outputs[0]["score"]["global"]["score"])
+
+
+    def test_detector(self):
+        metric = Detector(model_name = "MilaNLProc/bert-base-uncased-ear-misogyny")
+        predictions = ["I hate women.", "I do not hate women."]
+        references = [["I hate women."], ["I do not hate women."]]
+        outputs = apply_metric(
+            metric=metric, predictions=predictions, references=references
+        )
+        
+        global_target = 0.9562818706035614
         self.assertAlmostEqual(global_target, outputs[0]["score"]["global"]["score"])
```

Comment:
You got here this error:
```
    self.assertAlmostEqual(global_target, outputs[0]["score"]["global"]["score"])
AssertionError: 0.9562818706035614 != 0.9562810361385345 within 7 places (8.344650268554688e-07 difference)
```
I think an appropirate fix will be to check within 4 decimal points: 
```suggestion
        self.assertAlmostEqual(global_target, outputs[0]["score"]["global"]["score"], places=4)
```

Reply:
thank you - also fixed in another commit!